thunder/tests/test_fa3_executor.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import torch
+
+import thunder
+import thunder.core
+from thunder.executors.fa3ex import fa3_ex
```

Comment:
```
from thunder.executors.fa3ex import fa3ex or fas3_executor?
```

Reply:
I copied this off test_sdpaex_executor.py following the import pattern at https://github.com/Lightning-AI/lightning-thunder/blob/d202ba3d868642e0b4d61cd1bf93794350d5a663/thunder/tests/test_sdpaex_executor.py#L6 and subsequent usage pattern at https://github.com/Lightning-AI/lightning-thunder/blob/d202ba3d868642e0b4d61cd1bf93794350d5a663/thunder/tests/test_sdpaex_executor.py#L87, this should be correct